### PR TITLE
See more less article

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/assets/javascripts/new_feeds.js [hunk 1]
```diff
@@ -31,4 +31,8 @@ $(document).on("turbolinks:load ajaxComplete", function() {
       $.get("/share_posts/new", {share_post: {post_id: post_id}});
     });
   }
+
+  $(".post-article").readmore({
+    lessLink: "<a href='#'>Read Less</a>"
```
Comment (NgengChhengkim) [inline app/assets/javascripts/new_feeds.js]: ort brer I18n knong javascript te?

File: app/views/shared/_new_feed.html.erb [hunk 2]
```diff
@@ -58,13 +58,15 @@
...
       <% end %>
     </span>
-    <span class="share">
-      <%= link_to "javascript:void(0)", class: "share-post-action",
-        data: {post_id: post.id} do %>
-        <i class="fa fa-share" aria-hidden="true"></i>
-        <%= t "new_feeds.reacts.share" %>
-      <% end %>
-    </span>
+    <% unless activity.user_id == current_user.id %>
```
Comment (NgengChhengkim) [inline app/views/shared/_new_feed.html.erb]: user share post kloun eng ban ta.

File: app/assets/javascripts/new_feeds.js (hunk 1 above)
Comment (yongsokheng) [reply app/assets/javascripts/new_feeds.js]: ot brer gem i18n js pong

File: app/views/shared/_new_feed.html.erb (hunk 2 above)
Comment (yongsokheng) [reply app/views/shared/_new_feed.html.erb]: doch ot ban te :smile:

**Pull request merged**